packages/components/organisms/f-footer/src/components/LegalField.vue

```diff
@@ -64,5 +74,9 @@ export default {
     width: 50px;
     height: 50px;
 }
+
+.c-legalField-certificates-link:focus {
+    @extend %u-elementFocus--borderless
+}
```

Comment:
Some browsers prefer `:focus-visible` so we should specify both. Also semicolon.
```suggestion
.c-legalField-certificates-link {
    &:focus,
    &:focus-visible {
        @extend %u-elementFocus--borderless;
    }
}
```